docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-recording-rule.md

```diff
@@ -1,38 +1,38 @@
 +++
-title = "Create Cortex or Loki managed recording rule"
-description = "Create Cortex or Loki managed recording rule"
+title = "Create Grafana Mimir or Loki managed recording rule"
+description = "Create Grafana Mimir or Loki managed recording rule"
 keywords = ["grafana", "alerting", "guide", "rules", "recording rules", "create"]
 weight = 400
 +++
 
-# Create a Cortex or Loki managed recording rule
+# Create a Mimir or Loki managed recording rule
 
-You can create and manage recording rules for an external Cortex or Loki instance. Recording rules calculate frequently needed expressions or computationally expensive expressions in advance and save the result as a new set of time series. Querying this new time series is faster, especially for dashboards since they query the same expression every time the dashboards refresh.
+You can create and manage recording rules for an external Mimir or Loki instance. Recording rules calculate frequently needed expressions or computationally expensive expressions in advance and save the result as a new set of time series. Querying this new time series is faster, especially for dashboards since they query the same expression every time the dashboards refresh.
```

Comment:
```suggestion
You can create and manage recording rules for an external Grafana Mimir or Loki instance. Recording rules calculate frequently needed expressions or computationally expensive expressions in advance and save the result as a new set of time series. Querying this new time series is faster, especially for dashboards since they query the same expression every time the dashboards refresh.
```